independent-projects/arc/tests/src/test/java/io/quarkus/arc/test/interceptors/targetclass/VoidAroundInvokeOnTargetClassTest.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.arc.test.interceptors.targetclass;
+
+import jakarta.inject.Singleton;
+import jakarta.interceptor.AroundInvoke;
+import jakarta.interceptor.InvocationContext;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.arc.Arc;
+import io.quarkus.arc.ArcContainer;
+import io.quarkus.arc.test.ArcTestContainer;
+
+public class VoidAroundInvokeOnTargetClassTest {
+    @RegisterExtension
+    public ArcTestContainer container = new ArcTestContainer(MyBean.class);
+
+    @Test
+    public void test() {
+        ArcContainer arc = Arc.container();
+        MyBean bean = arc.instance(MyBean.class).get();
+        bean.doSomething();
+    }
+
+    @Singleton
+    static class MyBean {
+        String doSomething() {
+            return "foobar";
+        }
+
+        @AroundInvoke
+        void intercept(InvocationContext ctx) throws Exception {
```

Comment:
I don't think that we should support this signature. According to the spec, only `Object <METHOD>(InvocationContext)` is supported: https://jakarta.ee/specifications/interceptors/2.0/interceptors-spec-2.0.html#business_method_interceptor_methods

Reply:
In other words, this particular test should result in a validation failure...

Reply:
If it wasn't reported as a failure before, we at least need to isolate this change as we probably wouldn't backport it.

Reply:
I'll see what would it take to fix the validation. I can either add a new commit to this PR, or submit another PR -- I guess another PR would be easier from the backporting perspective.

Reply:
:+1: Just remove this test pls.

Reply:
Fair enough. Rebased and removed this test.

Reply:
Thanks.